server/cluster.go

```diff
@@ -452,6 +453,13 @@ func (c *RaftCluster) collectMetrics() {
 	}
 	statsMap.Collect()
 
+	regionStats := newRegionStatistics(c.cachedCluster.opt, c.coordinator.classifier)
+	for _, r := range cluster.GetRegions() {
+		regionStats.Observe(r)
+	}
```

Comment:
Can you test or evaluate the performance cost here if we have 1 million regions?

Reply:
I need change to another method, GetRegions got poor performance.